crates/forge/tests/e2e/gas_report.rs

```diff
@@ -0,0 +1,117 @@
+use crate::e2e::common::runner::{setup_package, test_runner};
+use indoc::indoc;
+use shared::test_utils::output_assert::assert_stdout_contains;
+
+#[test]
+fn basic() {
+    let temp = setup_package("simple_package");
+    let output = test_runner(&temp)
+        .arg("--gas-report")
+        .arg("call_and_invoke")
+        .assert()
+        .code(0);
+
+    assert_stdout_contains(
+        output,
+        indoc! {r"
+    [..]Compiling[..]
+    [..]Finished[..]
+
+
+    Collected 1 test(s) from simple_package package
+    Running 0 test(s) from src/
+    Running 1 test(s) from tests/
+    [PASS] simple_package_integrationtest::contract::call_and_invoke (l1_gas: ~0, l1_data_gas: ~[..], l2_gas: ~[..])
+    ╭------------------------+-------+-------+-------+---------+---------╮
+    | HelloStarknet Contract |       |       |       |         |         |
+    +====================================================================+
+    | Function Name          | Min   | Max   | Avg   | Std Dev | # Calls |
+    |------------------------+-------+-------+-------+---------+---------|
+    | get_balance            | 13340 | 13340 | 13340 | 0       | 2       |
+    |------------------------+-------+-------+-------+---------+---------|
+    | increase_balance       | 24940 | 24940 | 24940 | 0       | 1       |
+    ╰------------------------+-------+-------+-------+---------+---------╯
+
+    Tests: 1 passed, 0 failed, 0 ignored, [..] filtered out
```

Comment:
👍 👍 👍

Reply:
@ddoktorski wdyt about having the contract name centered and the heading cells being merged into one?